nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/main.css

```diff
@@ -286,45 +290,125 @@ span.details-title {
     height: 28px;
 }
 
-#template-file-field {
+#select-file-button button {
+    position: absolute;
+    float: right;
+    right: 4px;
+    top: 22px;
+    font-size: 18px;
+    color: #004849;
+    border: none;
+    background-color: transparent;
+}
+
+/*#select-file-button button:hover {*/
```

Comment:
Should this commented style be retained, or can it be removed?

Reply:
Removed.